### ISSUE TITLE
Show visible locations

### ISSUE THREAD

**Issue by MattEqualsCoder** [Vivelin/SMZ3Randomizer#180]
Most tracker offer the ability to see what locations you can peak at. We should probably do that too.

I think this would be pretty simple. We could add something similar to the canAccess delegate, but it would default to false. For the map, I think we could probably mark it as a blue triangle or something to be different from the other location icons. Not sure how we would represent this in the location list.

**Comment by Vivelin** [comment]
I'm not sure if people even use the location list anymore outside of marked items. We could probably remove it entirely and show marked items on the map instead.

**Comment by TheRealFragger** [comment]
I agree, I think it should be there though but maybe as an optional thing? Like right click on the map or have a button on it to launch the location list. As for how to represent it, I was thinking of an eye icon in the same position as a key could be

**Comment by Vivelin** [comment]
We could probably show something in the marked locations list, actually. If something is in logic for peeking (`canPeek` or something) show it with some default/unknown icon.
